.travis.yml

```diff
@@ -28,7 +27,7 @@ script:
   - ansible-playbook -i tests/docker/hosts -e image_name=${IMAGE_NAME} tests/docker/site.yml
 
   # Idempotence test
-  - ansible-playbook -i tests/docker/hosts -e image_name=${IMAGE_NAME} tests/docker/site.yml > idempotence_out
+  - ansible-playbook -i tests/docker/hosts -e image_name=${IMAGE_NAME} tests/docker/site.yml | tee idempotence_out
```

Comment:
Some of the idempotence tests were failing due to Travis's "10min no output" policy, and there was no way to see what the delay was.  This allows full visibility into what's happening.